src/awx_plugins/inventory/plugins.py

```diff
@@ -1,5 +1,5 @@
 # FIXME: the following violations must be addressed gradually and unignored
-# mypy: disable-error-code="assignment, no-untyped-call, no-untyped-def, var-annotated"
+# mypy: disable-error-code='assignment, no-untyped-call, no-untyped-def, var-annotated'
```

Comment:
I'm assuming a misconfigured code editor made a bunch of invasive changes, eh?

Reply:
I ran black on this before I committed and it reformatted everything- I am going to pull out those changes today!

Reply:
Why didn't you run the automation that is integrated into the repo instead?

Reply:
Totally understandable :)
I was just configuring my editor for the new laptop the other day and the first thing I did was figuring out how to make it non-invasive because the framework I'm on happens to have formatting on save which is a harmful assumption to have universally in context of arbitrary projects I might want to open..